redis/client.py

```diff
@@ -392,7 +403,7 @@ def parse_georadius_generic(response, **options):
     if not options['withdist'] and not options['withcoord']\
             and not options['withhash']:
         # just a bunch of places
-        return [nativestr(r) for r in response_list]
+        return [text_or_binary(r) for r in response_list]
```

Comment:
We can just return the `response_list` here since it should be decoded or not based on the `decode_responses` flag.

Reply:
Ah, magic! That does do the correct thing.